src/actions/registerAction.js

```diff
@@ -0,0 +1,9 @@
+import { REGISTER_USER } from './types';
+
+export const registerAccount = (values) => dispatch => {
+  // TODO - Log in the user here.
+  console.log("meow")
```

Comment:
console.log the values to see if they print out here. If they do, we can just submit a post request with the values